DirectEmployers/MyJobs
# [PD-2536] wizard submission logic

This leaves NUO still buggy but able to, under limited circumstances, convert an outreach record to a communication record.

Unfortunately GitHub isn't picking up that I renamed process-email-* to process-outreach-*.

## Test

1. Make sure you have an outreach record to work with.
1. Pick an outreach record.
1. Pick an existing partner.
1. Pick an existing contact.
1. Fill in some data for the communication record.
1. Submit.
1. Using the shell and the outreach record ID you can see that the outreach record is in reviewed state and there is a new communication record.
